scripts/make_orog.py

```diff
@@ -71,7 +71,7 @@ def make_orog(config_file, key_path):
     logging.info(f"Will run make_orog in {task_rundir}")
 
     # The experiment config will have {{ CRES | env }} expressions in it that need to be
-    # dereferenced during driver initialization.
```

Comment:
Standardize punctation for comments in runscript.

Reply:
Hmm, I'm not really sure. I was standardizing the docstrings in the runscript and then the `parse_args` `help` / `message` descriptions and saw this difference at the same time. I didn't think about some of the comments being sentences vs. others not. I'm happy to add them back in on both the scripts if we'd like to keep that sentence structure.

Reply:
I think your changes are fine as-is at this point. :)